go/go.mod

```diff
@@ -4,7 +4,15 @@ go 1.15
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/gogo/protobuf v1.1.1
+	github.com/golang/snappy v0.0.2
 	github.com/gorilla/mux v1.8.0
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0 // indirect
```

Comment:
We're getting a deprecation warning as of this dependency:

```
WARNING: Package "github.com/golang/protobuf/protoc-gen-go/generator" is deprecated.
	A future release of golang/protobuf will delete this package,
	which has long been excluded from the compatibility promise.
```

Background here, in this comment (and thread): https://github.com/golang/protobuf/issues/1104#issuecomment-619488848